core/src/main/java/hudson/model/Executor.java

```diff
@@ -870,14 +852,10 @@ public HttpResponse doStop() {
     }
 
     /**
-     * Throws away this executor and get a new one.
+     * @deprecated now a no-op
      */
-    @RequirePOST
```

Comment:
I would not remove RequirePOST in such case, because it only makes the deprecated method more available 🐛